bot/bot.js

```diff
@@ -6,6 +6,9 @@ const Discord = require("discord.js");
 let config = require("config");
 config = config.get("bot");
 
+//load modules
+const claimbot = require('./modules/claimbot.js');
```

Comment:
There is no need to require plugins, as we have a plugin loader that will load them automatically if they exist in the modules folder